apiv6/go.mod

```diff
@@ -3,7 +3,6 @@ module github.com/kentik/community_sdk_golang/apiv6
 go 1.15
 
 require (
-	github.com/gorilla/handlers v1.5.1
```

Comment:
Just out of curiosity, is this change related to the ticket? I do not see removing usage of this package in those changes.

Reply:
No, I have just run `go mod tidy`